app/graph/types/project_media_type.rb

```diff
@@ -152,6 +152,14 @@
     }
   end
 
+  field :translations_count do
+    type types.Int
+
+    resolve ->(project_media, _args, _ctx) {
+      project_media.get_annotations('translation').count
+    }
+  end
```

Comment:
Could it be adding annotation_type as parameter in this field? https://github.com/meedan/check-api/blob/7a2c94679228dcf0f3295ecba46ac8a5f557f797/app/graph/types/project_media_type.rb#L86-L92